src/back/server.py

```diff
@@ -277,6 +220,4 @@ async def process_frame_for_skeleton(frame):
     app.router.add_options("/broadcast", broadcast)
     app.router.add_post("/consumer", consumer)
```

Comment:
please name the broadcast endpoint and the consume endpoint in a consistent manner

Reply:
will resolve in another issue because this require both frontend and backend changes